easybuild/easyconfigs/l/LAMMPS/LAMMPS-22Aug2018-CrayGNU-18.08-deepmd-0.10.2.eb

```diff
@@ -0,0 +1,45 @@
+# Contributed by Luca Marsella and Victor Holanda Rusu (CSCS)
+easyblock = 'MakeCp'
+
+name = 'LAMMPS'
+version = '22Aug18'
+release = 'stable_22Aug2018'
+deepmd_version = '0.10.2'
+versionsuffix = '-deepmd-%s' % deepmd_version
+
+homepage = 'http://lammps.sandia.gov/'
+description = "LAMMPS (Large-scale Atomic/Molecular Massively Parallel Simulator) is a classical molecular dynamics simulation code designed to run efficiently on parallel computers."
+
+toolchain = {'name': 'CrayGNU', 'version': '18.08'}
+toolchainopts = { 'usempi': True, 'openmp': True }
+
+source_urls = ['https://github.com/lammps/lammps/archive']
+sources = ['%s.tar.gz' % (release)]
+
+prebuildopts = ' cd ./src && '
+prebuildopts += ' cp -r $EBROOTDEEPMD/USER-DEEPMD . && '
+prebuildopts += ' export CPATH=$EBROOTDEEPMD/include/deepmd:$CPATH && '
+prebuildopts += ' make yes-standard yes-user-cg-cmm yes-user-omp yes-user-misc yes-user-deepmd && '
+prebuildopts += ' make no-voronoi no-reax no-poems no-meam no-kim no-kokkos no-mscg no-gpu no-latte && '
+prebuildopts += ' make package-update && '
+# go to folder ./lib/reax and make package reax
+prebuildopts += ' pushd ../lib/reax && make -f Makefile.gfortran && popd && '
+# create Makefile.omp and correct Makefile.mpi
+prebuildopts += ' sed -e \'s/mpicxx/CC -fopenmp/\' ./MAKE/Makefile.mpi > ./MAKE/Makefile.omp && '
+prebuildopts += ' sed -i -e \'s/mpicxx/CC/\' ./MAKE/Makefile.mpi && '
+buildopts = [ ' mpi ', ' omp ' ]
+
+builddependencies = [
+    ('cray-fftw/3.3.6.2', EXTERNAL_MODULE),
+    ('cray-python/2.7.13.1', EXTERNAL_MODULE),
```

Comment:
Please use the standard for Python modules in production builds:
https://github.com/eth-cscs/production/wiki/UES-standards-(CUDA,-Python,-...)-for-EB-recipes